src/utils/objectFormatter.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2020 The Ontario Institute for Cancer Research. All rights reserved
```

Comment:
Only commenting on the `2020` here because this a new file 😛

Reply:
lol thx

Reply:
```suggestion
 * Copyright (c) 2022 The Ontario Institute for Cancer Research. All rights reserved
```
fixed that for you ;) lol